### PR TITLE
Modify rule S6147: refactor directory structure, set quickfix and code-snippet language

### PR DESCRIPTION
**Pull request by necto**
I recommend review by commit

### CODE REVIEW

File: rules/S6147/cfamily/rule.adoc [hunk 1]
```diff
@@ -1 +1,110 @@
-include::../rule.adoc[]
+== Why is this an issue?
```
Comment (pdschbrt) [inline rules/S6147/cfamily/rule.adoc:1]: Aren't we missing a one-liner title here?

**Review by pdschbrt — CHANGES_REQUESTED** [review]
It seems you only moved stuff around and fixed the encoding.

However, I think we lost the title while doing the clean-up. Please recover it.

File: rules/S6147/cfamily/rule.adoc (hunk 1 above)
Comment (necto) [reply rules/S6147/cfamily/rule.adoc:1]: AFAIK, no we are not.
The one-liner title comes from metadata.json, and it is added during rendering on top by the rspec frontend page for convenience.

Comment (necto) [reply rules/S6147/cfamily/rule.adoc:1]: I simply inlined this `include::[]` directive, copying verbatim the (previously redacted) content of `../rule.adoc`

Comment (pdschbrt) [reply rules/S6147/cfamily/rule.adoc:1]: I may have misremembered this. Thanks for double-checking.

**Review by pdschbrt — APPROVED** [review]

**Pull request merged**